src/MatrixTools.js

```diff
@@ -24,5 +25,34 @@ module.exports = {
     getDisplayAliasForRoom: function(room) {
         return room.getCanonicalAlias() || room.getAliases()[0];
     },
+
+    isDirectMessageRoom: function(room, me, ConferenceHandler, hideConferenceChans) {
```

Comment:
Yep, made it a separate function, as you mentioned this when I was doing it, so this should be easily swapped out for the new behaviour once it's implemented.